src/locales/en_US.json

```diff
@@ -593,6 +594,343 @@
     "alternate": "Alternate",
     "complex": "Complex (A / B)"
   },
+  "outcome": {
+    "core": [
+      {
+        "id": "a-couriers-job",
+        "name": "A courier's job",
+        "effect": ""
+      },
+      {
+        "id": "adamant",
+        "name": "Adamant",
+        "effect": ""
+      },
+      {
+        "id": "a-hideous-act",
+        "name": "A hideous act",
+        "effect": ""
+      },
+      {
+        "id": "a-lovers-request",
+        "name": "A lover's request",
+        "effect": ""
+      },
+      {
+        "id": "corrupted-hero",
+        "name": "Corrupted hero",
+        "effect": ""
+      },
+      {
+        "id": "curse-breaker",
+        "name": "Cursebreaker",
+        "effect": ""
+      },
+      {
+        "id": "dynamic-duo",
+        "name": "Dynamic duo",
+        "effect": ""
+      },
+      {
+        "id": "gentle-dismissal",
+        "name": "Gentle dismissal",
+        "effect": ""
+      },
+      {
+        "id": "guardians-curse",
+        "name": "Guardian's curse",
+        "effect": "As long as you have Guardian's Curse, you cannot unequip the Cursed Bracelet."
+      },
+      {
+        "id": "humbled",
+        "name": "Humbled",
+        "effect": ""
+      },
+      {
+        "id": "lovebirds-reunited",
+        "name": "Lovebirds reunited",
+        "effect": ""
+      },
+      {
+        "id": "relic-breaker",
+        "name": "Relic breaker",
+        "effect": ""
+      },
+      {
+        "id": "sowing-winds",
+        "name": "Sowing winds",
+        "effect": ""
+      },
+      {
+        "id": "storyteller",
+        "name": "Storyteller",
+        "effect": ""
+      },
+      {
+        "id": "unbroken",
+        "name": "Unbroken",
+        "effect": ""
+      }
+    ],
+    "apocalypse": {
+      "campaign": [
```

Comment:
Better to just keep it the same structure for all campaigns.

So if we introduce a array with campaign and storyrecord keys we should keep it consistent for the core and awakenings campaigns as well.

Reply:
Otherwise looking good 👍

Reply:
So, if I undestand you correctly campaign and storyrecord should be always included? Even though for example there are  no story record for a specific key type / version?

Reply:
It's a minor chance but keeps it consistent

Reply:
Because currently I am preparing to include the other keywords. And it looks currently like that:

`
"auras":{
       "core":[
           ...
        ],
       "apocalypse":[
           ...
        ],
        "awakenings":[
        ],
},

"statuses":{
       "core":[
           ...
        ],
       "apocalypse":{
         "campaign":[
           ...
           ],
         "storyrecord":[
           ...
           ]
         },
        "awakenings":[
           ...
           ],
         "storyrecord":[
           ...
           ]
         }
},

`

Reply:
It should be consistent, yeah. No problem, a minor change. Will update it tomorrow I think.

Reply:
I updated the json files and repos.